docs/README.md

```diff
@@ -11,13 +11,13 @@ This is the main Iroha 2 documentation that you will find useful:
 
 Documentation for Iroha 2 tools:
 
-- [Kagami](../tools/kagami/README.md)
-- [Kura Inspector](../tools/kura_inspector/README.md)
-- [Parity Scale Decoder Tool](../tools/parity_scale_cli/README.md)
+- [Kagami](../crates/iroha_kagami/README.md)
+- [Kura Inspector](../crates/kura_inspector/README.md)
+- [Parity Scale Decoder Tool](../crates/iroha_codec/README.md)
```

Comment:
I think we might want to have tools in a tools directory

Reply:
What's the benefit? They're still just plain old crates

Reply:
I think it only does so for tools used in CI, while Iroha tools are still part of Iroha

Reply:
#5043